reporters/CLIReporter.js

```diff
@@ -17,7 +17,7 @@ class CLIReporter extends BaseReporter {
         this.alwaysLog(chalk.cyan(`URLs to crawl: ${options.urls}`));
 
         // show progress bar only if we are not printing all logs to screen (verbose)
-        this.progressBar = (this.verbose || options.urls === 0) ? null : new ProgressBar('[:bar] :percent ETA :etas fail :fail% :site', {
```

Comment:
To be fair, "verbose" output is already messy because there's a lot of warnings.
I mainly added this here for the ETA, which is pretty useful, but it's fairly simple to do without a progress bar, I'll change it.